backend/msgpack/conn.go

```diff
@@ -60,7 +64,7 @@ type connection struct {
 	numFailures int
 	threshold   int
 
```

Comment:
Sounds good to me. To be honest, I never really had an arrangement myself, but I really like this one, will use it moving forward.